pkg/api/graph/interfaces.go

```diff
@@ -38,7 +38,7 @@ const (
 type Markers []Marker
 
 // MarkerScanner is a function for analyzing a graph and finding interesting things in it
-type MarkerScanner func(g Graph, f Namer) []Marker
+type MarkerScanner func(g Graph, f Namer, chaseRefs bool) []Marker
```

Comment:
> @deads2k I don't like this but do you have any better idea for how to pass info down to the marker level that --chase-refs is being used?

This implies that the marking process is chasing references?  I don't think that's the right place to continue making API calls.  Having a marking stage mutate your graph is going to cause problems, but having different markers inspect a different graph (even if you never persist it, that's what you're doing) is also hard to reason about.

I think I'd rather have a coarse grained, "hide this marker" and then a multi-namespace `oc status` than try to sometimes chase in a marking function.